composer.json

```diff
@@ -11,7 +11,8 @@
         "drupal/migrate_plus": "^6.0",
         "php-http/message": "^1",
         "php-http/message-factory": "^1",
-        "guzzlehttp/psr7": "~2.6.2"
+        "guzzlehttp/psr7": "~2.6.2",
+        "drupal/datetime_more": "^1.0"
```

Comment:
@g2410g Please add this with the other drupal requires.

Reply:
@gargsuchi Updated the code.